Sources/HaCWebsiteLib/ViewModels/Script.swift

```diff
@@ -0,0 +1,68 @@
+import HaCTML
+import DotEnv
+import Foundation
+
+protocol JavaScriptable {
+  var javaScript: String {get}
+}
+
+extension String: JavaScriptable {
+  var javaScript : String {
+    // TODO: PROPERLY ESCAPE THIS! (eg. conver newlines to backslash)
+    // NOTE: A JSON LIBRARY COULD BE USED FOR THIS!
+    return "\"\(self)\""
```

Comment:
This needs to be done before merge!

Reply:
Does this still need doing @moosichu ?

Reply:
Yes, I'm working on it for another issue though.